src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkWatcherAPITests.ps1

```diff
@@ -703,9 +703,6 @@ function Test-ProvidersList
         Assert-AreEqual $list1.Countries.CountryName "United States"
         Assert-AreEqual $list2.Countries.CountryName "United States"
         Assert-AreEqual $list2.Countries.States.StateName "washington"
-        Assert-AreEqual $list3.Countries.CountryName "United States"
```

Comment:
@irrogozh since we brought back the command that populates the value of `$list3`, can we bring back these checks?